src/Blocks/custom/carousel/carousel.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+/**
+ * Template for the Carousel Block.
+ *
+ * @package Unicorns
+ */
+
+use UnicornsVendor\EightshiftLibs\Helpers\Components;
+use Unicorns\Manifest\Manifest;
+
+$manifest = Components::getManifest(__DIR__);
+
+$blockClass = Components::checkAttr('blockClass', $attributes, $manifest);
+$blockJsClass = Components::checkAttr('blockJsClass', $attributes, $manifest);
+$isLoop = Components::checkAttr('isLoop', $attributes, $manifest);
+$showItems = Components::checkAttr('showItems', $attributes, $manifest);
+
+$carouselClass = Components::classnames([
+	$blockClass,
+	$blockJsClass,
+	'swiper-container',
+]);
+
+?>
+
+<div class="<?php echo esc_attr($carouselClass); ?>" data-swiper-loop="<?php echo esc_attr($isLoop); ?>" data-show-items="<?php echo esc_attr($showItems); ?>">
+
+	<div class="<?php echo esc_attr('swiper-wrapper'); ?>">
+		<?php echo wp_kses_post($innerBlockContent); ?>
+	</div>
+
+	<div class="<?php echo esc_attr('control-container'); ?>">
+		<div class="<?php echo esc_attr('js-block-carousel-prev-arrow'); ?>">
+			<img src="<?php echo \apply_filters(Manifest::MANIFEST_ITEM, 'arrow-left.svg') ?>" alt="Previous">
```

Comment:
why adding additional request with the image to load svg code. Put the SVG image in the blocks manifest and load it from there in Js and PHP part check this reference

https://github.com/infinum/eightshift-frontend-libs/blob/develop/blocks/init/src/Blocks/components/accordion/manifest.json